.github/workflows/ci.yml

```
@@ -54,7 +54,12 @@ jobs:
       - run: julia -e 'using InteractiveUtils; versioninfo(verbose=true)'
       - uses: julia-actions/cache@v2
       - uses: julia-actions/julia-buildpkg@v1
-      - uses: julia-actions/julia-runtest@v1
+      - name: Run single-threaded tests
+        uses: julia-actions/julia-runtest@v1
+      - name: Run multithreaded tests
+        uses: julia-actions/julia-runtest@v1
+        env:
+          JULIA_NUM_THREADS: 2
```

```suggestion
      - name: Run multithreaded tests
        uses: julia-actions/julia-runtest@v1
        with:
          coverage: false
        env:
          JULIA_NUM_THREADS: 2
```

Most tests seem to be failing in the coveralls upload. Maybe the multithreading is messing things up somehow? Try to deactivate it for the multithreading tests to see if it fixes things.